packages/salesforcedx-vscode-apex/test/vscode-integration/commands/apexTestSuite.test.ts

```diff
@@ -10,40 +10,37 @@ import { ContinueResponse } from '@salesforce/salesforcedx-utils-vscode';
 import { expect } from 'chai';
 import { createSandbox } from 'sinon';
 import * as vscode from 'vscode';
+import {
+  ApexTestQuickPickItem,
+  TestType
+} from '../../../src/commands/apexTestRun';
 import {
   ApexTestSuiteOptions,
   TestSuiteBuilder,
   TestSuiteCreator,
   TestSuiteSelector
 } from '../../../src/commands/apexTestSuite';
-import {
-  ApexTestQuickPickItem,
-  TestType
-} from '../../../src/commands/forceApexTestRun';
 import { workspaceContext } from '../../../src/context';
 
 const sb = createSandbox();
 
-describe('Test Suite Selector', async () => {
+describe('Test Suite Selector', () => {
   let quickPickStub: sinon.SinonStub;
-  let retrieveSuitesStub: sinon.SinonStub;
 
-  beforeEach(async () => {
+  beforeEach(() => {
     sb.stub(workspaceContext, 'getConnection');
     quickPickStub = sb.stub(vscode.window, 'showQuickPick').returns({
       label: 'SuiteOne',
       type: TestType.Suite
     });
 
-    retrieveSuitesStub = sb
-      .stub(TestService.prototype, 'retrieveAllSuites')
-      .resolves([
-        { TestSuiteName: 'SuiteOne', Id: 'xxxxxxxxxxx27343' },
-        { TestSuiteName: 'SuiteTwo', Id: 'xxxxxxxxxxx27343' }
-      ]);
+    sb.stub(TestService.prototype, 'retrieveAllSuites').resolves([
```

Comment:
there is an async function here. Looks like it is not adequate to delete async keyword.